packages/grpc-js/src/subchannel.ts

```diff
@@ -298,8 +298,6 @@ export class Subchannel {
           return checkServerIdentity(sslTargetNameOverride, cert);
         };
         connectionOptions.servername = sslTargetNameOverride;
-      } else {
-        connectionOptions.servername = getDefaultAuthority(this.channelTarget);
```

Comment:
This does not seem to be in scope of this PR?

Reply:
That change is necessary because of the `dns-resolver` change.